src/main/java/com/metamx/emitter/core/factory/EmitterFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.metamx.emitter.core.factory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.metamx.common.lifecycle.Lifecycle;
+import com.metamx.emitter.core.Emitter;
+import com.metamx.http.client.HttpClient;
+
+public interface EmitterFactory
```

Comment:
Alternatively the "new" one could be completely built from json in the property instead of trying to construct a map then creating a json off of it. 

Either way, this allows anyone to write a json emitter that can be deserialized by the `ObjectMapper`